project.clj

```diff
@@ -76,7 +76,7 @@
 
   :plugins [[lein-release "1.0.5" :exclusions [org.clojure/clojure]]]
 
-  :uberjar-name "puppet-server-release.jar"
+  :uberjar-name "puppetserver-release.jar"
```

Comment:
@haus This PR updates the one reference to "puppet-server" to "puppetserver" in the "restarting.markdown" doc - https://github.com/puppetlabs/puppetserver/pull/1022/files/9d20775a6e446b457bc4827ffd79e6cdb450d6ad#r60331291.  Anything else you can think of besides that which would need updating?